### DISCUSSION TITLE
ARM Support

### DISCUSSION THREAD

**Discussion by pixelsquared** [Secure-Compliance-Solutions-LLC/GVM-Docker#243] [Feature Request]
I am looking into what it would take to add ARM support to the GVM-Docker and/or OpenVAS-Docker images.

If anyone has ideas on how we could do this or issues we would run into.

**Comment by Dexus** [comment]
This is not a big deal but then you don‘t have the support of all modules like openvas-smb.


But as far as I know everything else will be support ARM.
